src/imageproxy.rs

```diff
@@ -192,6 +192,17 @@ impl TryFrom<ImageProxyConfig> for Command {
     }
 }
 
+///BlobInfo collects known information about a blob
+#[derive(Debug, serde::Deserialize)]
+pub struct BlobInfo {
+    ///Diffid
```

Comment:
This comment could probably be expanded; something like
```suggestion
    /// Uncompressed digest of a layer; for more information, see https://github.com/opencontainers/image-spec/blob/main/config.md#layer-diffid
```

It's also more normal to have a space between the doc comment and text, i.e. `/// Size of blob` and not `///Size of blob` as below.